db/migrate/20160324174132_create_container_deployments.rb

```diff
@@ -0,0 +1,25 @@
+class CreateContainerDeployments < ActiveRecord::Migration[5.0]
+  def change
+    create_table :container_deployments do |t|
+      t.string     :deployment_type
+      t.string     :deployment_version
+      t.boolean    :containerized, :default => true
+      t.string     :deployment_method
+      t.string     :ssh_private
+      t.string     :ssh_public
+      t.string     :ssh_user
+      t.string     :rhsm_user
+      t.string     :rhsm_pass
+      t.string     :rhsm_sku
+      t.string     :rhsm_pool_id
+      t.string     :rhsm_server
+      t.string     :rhsm_activation_key
+      t.string     :metrics_endpoint
+      t.text       :customize
```

Comment:
Not sure I like this column name.  customize is a verb, and columns are usually nouns.  What does this column hold?  Perhaps `customizations`?